projects/plugins/jetpack/_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -276,6 +277,7 @@ class JetpackNotices extends React.Component {
 						onDismissClick={ this.props.clearLicensingError }
 					/>
 				) }
+				<UserLicenseActivationNotice />
```

Comment:
No, actually we want this to apply to everyone. (although the notice will only appear if a user has an unattached "user" license). 

The `jetpack_licensing_ui_enabled` filter is specifically for the licensing UI used for 'Partner' licenses, however we will be using a different licensing UI specifically for "user" (not partner) licenses, that has been created in #21551.  
I only used and enabled the `jetpack_licensing_ui_enabled` filter in the testing instructions to demonstrate what happens when submitting a license key (because it just so happens to work fine for attaching "user" licenses too).

Reply:
Thanks for the explanation!